docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst

```diff
@@ -16,17 +16,16 @@ Bootstrap
 Pilot3 needs a JSON file for bootstrapping. We simply call this file the ``pilot.json`` file.
 The ``pilot.json`` file is created starting from information found in the Configuration Service.
 
-The pilot wrapper (the script that starts the pilot, which is effectively equivalent to what SiteDirectors send) expects
-to find (download) such a ``pilot.json`` file from a known location, which can be for example exposed by the DIRAC
-WebApp webserver.
+The pilot wrapper (the script that starts the pilot, which is effectively equivalent to what SiteDirectors send)
+expects to find (download) such pilot.json file from a known location, or a set of them.
```

Comment:
```suggestion
expects to find and download such a ``pilot.json`` file from a known location, or a set of them.
```
if you just put download in parenthesis, then it should be "find ... at a known location"